tests/ui/conftest.py

```diff
@@ -14,53 +28,111 @@ def capabilities(capabilities):
     capabilities['marionette'] = True
     return capabilities
 
+
 @pytest.fixture
-def fxa_account(base_url):
-    url = DEV_URL if 'dev' in base_url else PROD_URL
+def fxa_account(our_base_url):
+    url = DEV_URL if 'dev' in our_base_url else PROD_URL
     return FxATestAccount(url)
 
 
-@pytest.fixture(scope='session')
-def jwt_issuer(base_url, variables):
+@pytest.fixture
+def jwt_issuer(our_base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_issuer']
+        hostname = urlparse.urlsplit(our_base_url).hostname
+        return json_file['api'][hostname]['jwt_issuer']
     except KeyError:
         return os.getenv('JWT_ISSUER')
 
 
-@pytest.fixture(scope='session')
-def jwt_secret(base_url, variables):
+@pytest.fixture
+def jwt_secret(our_base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_secret']
+        hostname = urlparse.urlsplit(our_base_url).hostname
+        return json_file['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
-def jwt_token(base_url, jwt_issuer, jwt_secret):
-    payload = {
-        'iss': jwt_issuer,
-        'iat': datetime.datetime.utcnow(),
-        'exp': datetime.datetime.utcnow() + datetime.timedelta(seconds=30)}
-    return jwt.encode(payload, jwt_secret, algorithm='HS256')
+def initial_data(transactional_db, live_server, our_base_url):
+    hostname = urlparse.urlsplit(our_base_url).hostname
+    # call_command('reset_db', interactive=False)
+    # call_command('syncdb', interactive=False)
+
+    call_command('loaddata', 'initial.json')
+
+    create_switch('super-create-accounts')
+
+    # call_command('import_prod_versions')
+    call_command('generate_addons', 1, app='firefox')
+    call_command(
+        'createsuperuser',
+        interactive=False,
+        username='uitest',
+        email='uitester@mozilla.org',
+        add_to_supercreate_group=True,
+        save_api_credentials='tests/ui/variables.json',
```

Comment:
Can we just add this to the `variables` fixture, rather than writing it to disk?